src/style.hpp

```diff
@@ -16,7 +16,7 @@ typedef std::vector<Vec2> Points;
 
 // The output for Distance Determination Algorithms (distance.hpp/cpp). Currently 
 // set to a floating point value that represents the distance from Earth
```

Comment:
Change this description for us, so its now just a Vec3 thing instead of a floating point value